index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name ="viewport" content="width= device-width, initial-scale = 1.0">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
+        <link rel="stylesheet" href="styles.css">
+        <title>Munana Clinton </title>
+     </head>
+     <body>
+        <header id="header">
+            <nav class="header-nav">
+                <h3 id="logo">Munana Clinton</h3>
+                <div class="menu-bar">
+                    <span class="nav-bar"></span>
+                    <span class="nav-bar"></span>
+                    <span class="nav-bar"></span>
+                </div>
+                
+                </nav>
+        </header>
+        <section class="headline">
+            <div class="headline-content">
+                <h1 id="intro-text">Hey There. Iam Alex</h1>
+                <h2 id="description-text">I'm a Software Developer</h2>
+                <h3 id="welcome-text">I can help you build a product , feature or website Look through some of my work and experience! 
+                    If you like what you see and have a project you need coded, 
+                    don’t hesitate to contact me.
+                </h3>  
+                <div class="my-socials">
+                    <a href="https://github.com/clin2on3mun"><img src="images/github.png"></a>
+                    <a href="https://www.linkedin.com/in/munana-clinton-b6122720b/"><img src="images/linkedIn.png"></a>
+                    <a><img src="images/angelist.png"></a>
+                    <a href="https://twitter.com/ClintonMunana"><img src="images/twitter.png"></a>
+                    <a><img src="images/Medium.png"></a>
```

Comment:
- [ ] Please add an `alt` attr for those `img` tags because it will make your website semantic and it will help user with screen readers